kubernetes/README.md

```diff
@@ -9,6 +9,16 @@ This example is _not_ production ready. In a production setting, you should:
 * Ensure your Docker daemon is accessible to the earthly pod
 * Mount / configure the workspace so you can get any artifacts produced out of the pod
 
+## Dependencies
+
+Requires [`kind`](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/) to run the automated tools. Follow their instructions for installation on your platform.
+
+## Running The Example
+
+Clone this repository, and then simply run `earthly ./kubernetes+start`. This will use `earthly`'s [`LOCALLY`](https://docs.earthly.dev/docs/earthfile#locally-experimental)functionality to set up a cluster, and build our [example project](https://github.com/earthly/ci-example-project) in it.
```

Comment:
```suggestion
Clone this repository, and then simply run `earthly ./kubernetes+start`. This will use `earthly`'s [`LOCALLY`](https://docs.earthly.dev/docs/earthfile#locally-experimental) functionality to set up a cluster, and build our [example project](https://github.com/earthly/ci-example-project) in it.
```